libvmaf/src/feature/cuda/integer_adm_cuda.c

```diff
@@ -32,6 +32,7 @@
 #include "picture_cuda.h"
 #include <unistd.h>
 #include <assert.h>
+#include "nvtx3/nvToolsExt.h"
```

Comment:
This header and the `nvtx*` calls need to go behind a `HAVE_NVTX` guard.

Reply:
Should be behind the guard now.